src/viur/core/modules/script.py

```diff
@@ -162,3 +163,40 @@ def update_path(self, skel):
             key = parent_skel["parententry"]
 
         skel["path"] = "/".join(path)
+
+    @exposed
+    def get_importable(self):
+
+        cuser = current.user.get()
+        # get importable key
+        qry_importable = self.viewSkel("node").all().filter("parententry", self.rootnodeSkel(ensure=True)["key"])
+        qry_importable = self.listFilter(qry_importable)
```

Comment:
```suggestion
        if not (qry_importable := self.listFilter(qry_importable)):
            raise errors.Unauthorized()
```
...and please test it!